File: Solutions/0561/0561.py
# -*- coding:utf-8 -*-
# Author: washing
# DateTime: 2021/2/16 11:35
# File: 0561.py
# Desc: 
class Solution:
    def arrayPairSum(self, nums: List[int]) -> int:
        nums.sort()
        return sum(nums[::2])

File: Solutions/0561/0561.java
/**
 * @Author: washing
 * @DateTime: 2021/02/16 11:34
 * @File: 0561.java
 * @Desc: 
 **/

class Solution {
    public int arrayPairSum(int[] nums) {
        Arrays.sort(nums);
        int count = 0;
        for (int i=0; i<nums.length; i+=2){
            count += nums[i];
        }
        return count;
    }
}